etc/kayobe/dnf.yml

```diff
@@ -41,7 +41,27 @@
 #     file: myrepo
 #     gpgkey: http://gpgkey
 #     gpgcheck: yes
-dnf_custom_repos:
+dnf_custom_repos: "{{ dnf_custom_repos_el8 | combine(lookup('vars', 'dnf_custom_repos_' ~ os_distribution )) }}"
```

Comment:
Might make sense to use `ansible_facts.distribution` here, if that works?

Reply:
Tested this and we do have access to facts during the plays that configure the dnf repos.